webapp/src/components/mobilitymap/Map.js

```diff
@@ -162,7 +164,13 @@ const Map = () => {
         <CardOverlay>
           <FormError error={error} noMargin component="div" />
           {readOnlyReason(user, "read_only_zero_balance") && (
-            <Link to="/funding">{t("common:add_money_to_account")}</Link>
+            <Alert className={clsx("mb-0", Boolean(error) && "mt-3")} variant="warning">
+              <p>
+                <i className="bi bi-exclamation-triangle-fill me-1"></i>
+                {t("mobility:add_funds_warning")}
+              </p>
+              <SumaButton href="/funding">{t("forms:add_funds")}</SumaButton>
```

Comment:
What is a 'suma button'? Why can't we use one of the existing button classes? I'm confused what is going on here.

Reply:
Not sure about the name either but Anne has commented about the use of different button styles on some of the app pages and advises to set a general button across the app. I'm trying to start that here. We could omit it for now
.